.github/workflows/build.yml

```diff
@@ -259,7 +259,7 @@ jobs:
           cache-from: type=local,src=${{ env.BUILDX_CACHE_DIR }}
           cache-to: type=local,dest=${{ env.BUILDX_CACHE_DIR }}
           context: .
-          file: ./Dockerfile
+          file: ./docker/api/Dockerfile
```

Comment:
Is there a reason why the location of this file has been changed from its location in the skeleton?

Reply:
We are planning on introducing an additional frontend container in the project and likely customizing the Mongo container as well so we anticipate referencing multiple Dockerfiles in this project. We would reference those Dockerfiles individually in `docker-compose.yml` and thought it made sense to keep those in the docker repo.

We discussed a few methods on how to do that cleanly and landed on this approach because it prevents renaming from `Dockerfile` standards for each and keeps them organized in a docker subdirectory. Research link for more info: https://stackoverflow.com/questions/27409761/docker-multiple-dockerfiles-in-project 

There are multiple ways to solve it, but we thought this approach was a good starting point. Our goal is to meet the standards and reduce complexity, so we are open to any feedback or suggestions for better approaches and appreciate any input.